### ISSUE TITLE
Require spaces around parenthesis in top file

### ISSUE THREAD

**Issue by clickthisnick** [warpnet/salt-lint#276]
**Is your feature request related to a problem? Please describe.**

We have noticed that compound matching does not work in the top.sls file if you don't have spaces around the parenthesis

```bash
  'G@apple:yum and G@spinach:yum and (G@name:foo or G@name:bar or G@name:baz)':
    - foo.bar
```

The above did not work, whereas below with spaces `( ` and ` )` worked.

```bash
  'G@apple:yum and G@spinach:yum and ( G@name:foo or G@name:bar or G@name:baz )':
    - foo.bar
```

**Describe the solution you'd like**

It would be nice if salt-lint gave a warning to include spaces here

**Describe alternatives you've considered**

Haven't thought about anything else 😉 

**Additional context**

None

**Comment by roaldnefs** [comment]
Thanks @clickthisnick for the feature request!

According to the [SaltStack documentation](https://docs.saltproject.io/en/latest/topics/targeting/compound.html#precedence-matching) the compound matchers indeed requires spaces between the parentheses and targets:

> Be certain to note that spaces are required between the parentheses and targets. Failing to obey this rule may result in incorrect targeting!
